views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js

```diff
@@ -92,11 +92,12 @@ define([
 
         var min = interaction.attr('minChoices'),
             max = interaction.attr('maxChoices'),
+            msg,
             choiceCount = _.size(interaction.getChoices()),
             minInstructionSet = false;
 
-        //if maxChoice = 1, use the radio gorup behaviour
-        //if maxChoice = 0, inifinite choice possible
+        //if maxChoice = 1, use the radio group behaviour
+        //if maxChoice = 0, infinite choice possible
         if(max > 1 && max < choiceCount){
```

Comment:
To be clear, you want the whole condition to be `if(max >= 1 && max < choiceCount){` I suppose? That would make a lot of sense